### PR TITLE
Add php7-apcu, the PHP7 APC user cache #5366

### PR DESCRIPTION
**Pull request by andypost**
That's update based on php5-apcu package

### CODE REVIEW

**Comment by andypost** [comment]
Looks build broken somehow
I used to install/compile this extension after patching pecl https://github.com/skilld-labs/docker-php/blob/master/php7/Dockerfile#L52

**Comment by andypost** [comment]
PR updated with ` --with-php-config=php-config7` now it works
probably it makes sense to update syntax to new format `s/_builddir/builddir`

File: testing/php7-apcu/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Carlo Landmeter <clandmeter@gmail.com>
+pkgname=php7-apcu
+_pkgreal=apcu
```
Comment (jirutka) [inline testing/php7-apcu/APKBUILD]: Please name it rather as `_pkgname`, `_pkgreal` has very short Levenshtein distance from `_pkgrel`. ;)

Comment (andypost) [inline testing/php7-apcu/APKBUILD]: That's strange suggestion because `_pkgreal` & `_pkgver` looks a standard all over php* packages

**Comment by andypost** [comment]
maybe better to update all packages to use more readable names? Looks that out of scope if adding new package

File: testing/php7-apcu/APKBUILD (hunk 1 above)
Comment (jirutka) [inline testing/php7-apcu/APKBUILD]: Hm, you’re right, and not even php* packages, but also a perl* packages. Actually there are only slightly more packages that uses `_pkgname`.

I’ll discuss it on IRC later. For now, let’s keep `_pkgreal`.

File: testing/php7-apcu/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Carlo Landmeter <clandmeter@gmail.com>
```
Comment (jirutka) [inline testing/php7-apcu/APKBUILD]: Is @clandmeter okay with being maintainer of this package?

Comment (clandmeter) [inline testing/php7-apcu/APKBUILD]: better to let @andypost claim maintainership (if he want to move it to community)

PS i don't think we have a standard for local based variables, until we do so it shouldn't matter what to use.

**Comment by andypost** [comment]
I changed Maintainer to myself and removed previous contributor cos this package is new

**Comment by jirutka** [comment]
Merged in e07019964b87b8d37c77bd529a3e3c1ad180ffd6, thanks!

**Comment by andypost** [comment]
Yay! so what is next steps to see this in testing?

**Comment by jirutka** [comment]
It should be in there in few minutes.

**Comment by andypost** [comment]
Thanx a lot for guidance, will review other php7 stack needed for Drupal development

**Pull request closed without merging**